src/tests/cmocka/test_pam_srv.c

```diff
@@ -4485,6 +4612,12 @@ int main(int argc, const char *argv[])
                                         pam_test_setup_2fa_single_and_service_prompt, pam_test_teardown),
         cmocka_unit_test_setup_teardown(test_pam_prompting_2fa_single_and_service_srv,
                                         pam_test_setup_2fa_single_and_service_prompt, pam_test_teardown),
+#ifdef BUILD_PASSKEY
+        cmocka_unit_test_setup_teardown(test_pam_prompting_passkey_interactive,
+                                        pam_test_setup_passkey_interactive_prompt, pam_test_teardown),
+        cmocka_unit_test_setup_teardown(test_pam_prompting_passkey_interactive_and_touch,
+                                        pam_test_setup_passkey_interactive_and_touch_prompt, pam_test_teardown),
+#endif
```

Comment:
To keep it tidy, you could add the call to these tests along with the other passkey tests.

Reply:
Fixed.